src/Updates/class-auto-updater.php

```diff
@@ -191,6 +193,9 @@ public function check_update( $option ) {
 			'plugin'      => $this->_path,
 			'url'         => $this->_url,
 			'slug'        => $this->_slug,
+			'icons' => array(
+				'2x' => $this->_update_icon,
```

Comment:
If this is empty for some reason, will it break anything? Or will WP just handle it gracefully?

Reply:
Currently, i am requiring this as adam wants icons for all things added going forward. If its an empty string it will not break anything afaik, will test